### PR TITLE
set info.reverse as true for varian FIDs

### PR DESCRIPTION
**Pull request by jobo322**
The idea is to pass information to the FFT filter in nmrium in order to reverse the data. It is done multiplying by -1 the imaginary data.

### CODE REVIEW

File: src/read/readVarian/getInfo.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
```
Comment (targos) [inline src/read/readVarian/getInfo.ts:1]: ?

File: src/read/readVarian/getInfo.ts [hunk 2]
```diff
@@ -7,24 +8,42 @@ export function getInfo(varianOutput: Fid1D) {
...
+      const { sw = 1, rfl = 1, rfp = 1 } = meta;
+      maybeAdd(
+        info,
+        'baseFrequency',
+        originFrequency - (sw / 2 - rfl + rfp) / 1e6,
+      );
+    } else {
+      maybeAdd(info, 'baseFrequency', originFrequency);
+    }
   }
```
Comment (targos) [inline src/read/readVarian/getInfo.ts:47]: It looks like nothing is tested in this block. How do you know your changes work?

File: src/read/readVarian/getInfo.ts [hunk 3]
```diff
@@ -41,6 +60,14 @@ export function getInfo(varianOutput: Fid1D) {
...
-  return { info, meta };
+  info.isFid = !originalMetadata.status.isSpectrum;
+  if (info.isFid) {
+    info.reverse = true;
+  }
+
+  return { info, meta: { ...meta, ...originalMetadata } };
+}
+
+function getFirstIfArray(data: any) {
```
Comment (targos) [inline src/read/readVarian/getInfo.ts:71]: Please don't use `any`. It's almost never the right choice

File: src/read/readVarian/getInfo.ts (hunk 2 above)
Comment (jobo322) [reply src/read/readVarian/getInfo.ts]: I was checking a varian file with the missing parameter, but I artificially removed the parameter in order to test the calculation.

**Pull request merged**